src/test/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/SplitNCigarReadsUnitTest.java

```diff
@@ -60,23 +104,20 @@ public void splitReadAtN() {
                 final TestManager manager = new TestManager(header);
 
                 GATKRead read = ReadClipperTestUtils.makeReadFromCigar(cigar);
-                SplitNCigarReads.splitNCigarRead(read, manager);
+                SplitNCigarReads.splitNCigarRead(read, manager,header,true);
                 List<OverhangFixingManager.SplitRead> splitReads = manager.getReadsInQueueForTesting();
                 final int expectedReads = numOfSplits+1;
                 Assert.assertEquals(splitReads.size(), expectedReads, "wrong number of reads after split read with cigar: " + cigar + " at Ns [expected]: " + expectedReads + " [actual value]: " + splitReads.size());
-                final List<Integer> readLengths = consecutiveNonNElements(read.getCigar().getCigarElements());
+                final int readLengths = consecutiveNonNElements(read.getCigar().getCigarElements());
```

Comment:
Can you change this variable to readLength since it looks like it's not plural anymore.